### PR TITLE
Bunch of minor fixes to the text of README

### PR DESCRIPTION
**Pull request by filipovskii**

### CODE REVIEW

**Comment by lahmatiy** [comment]
Changes for `instr'umenter` in one of links looks like an mistake

File: README.md [hunk 1]
```diff
@@ -34,34 +34,34 @@ You should include `component-inspector` script right **after** `Backbone` scrip
...
 ```
 
-If you use `Express` you can use [express-open-in-editor](https://github.com/lahmatiy/express-open-in-editor) to provide some url with necessary functionality.
+There is an `Express` middleware [express-open-in-editor](https://github.com/lahmatiy/express-open-in-editor), which is able to provide an URL with necessary functionality.
 
-Or if you don't depend on dev-server you could use [basisjs-tools](https://github.com/basisjs/basisjs-tools) that support that feature.
+In case you don't depend on dev-server, you could use [basisjs-tools](https://github.com/basisjs/basisjs-tools) that support that feature.
 
-Also you could implement your own implmentation for your dev-server using [open-in-editor](https://github.com/lahmatiy/open-in-editor) as basis.
+Roll your own implmentation for your dev-server using [open-in-editor](https://github.com/lahmatiy/open-in-editor).
```
Comment (lahmatiy) [inline README.md]: missed `e` in `implmentation`

**Comment by lahmatiy** [comment]
Do you mind to squash?

**Comment by filipovskii** [comment]
Not at all :)

**Comment by lahmatiy** [comment]
Thank you!

**Pull request merged**